crates/turborepo-lib/src/config.rs

```diff
@@ -508,6 +534,21 @@ impl TurborepoConfigBuilder {
         Ok(local_config)
     }
 
+    fn get_global_auth(&self) -> Result<ConfigurationOptions, Error> {
+        let global_auth_path = self.global_auth_path()?;
+        let mut contents = global_auth_path
+            .read_existing_to_string_or(Ok("{}"))
+            .map_err(|error| Error::FailedToReadConfig {
+                config_path: global_auth_path.clone(),
+                error,
+            })?;
+        if contents.is_empty() {
+            contents = String::from("{}");
+        }
+        let global_auth: ConfigurationOptions = serde_json::from_str(&contents)?;
```

Comment:
I think we want to only read `token` from these sources. Small chances, but if the Vercel auth file ever adds additional fields that have shared names we could be in a bad situation.